tests/test_backend.py

```diff
@@ -77,6 +77,17 @@ def test_on_start_configures_volume_normalization(spotify_mock, config):
     volume_normalization_mock.assert_called_once_with(False)
 
 
+def test_on_start_configures_private_session(spotify_mock, config):
+    session = spotify_mock.Session.return_value
+    private_session_mock = mock.PropertyMock()
+    type(session).private_session = private_session_mock
```

Comment:
Thanks for the fast response! Yeah, my bad, fixed in dfa3d3b272efe3de720bd0333a3972505eb9ac1d and the CI build is passing now.